lava-master/Dockerfile

```diff
@@ -57,6 +57,8 @@ RUN a2enmod proxy \
 
 COPY configs/tftpd-hpa /etc/default/tftpd-hpa
 
+RUN git clone https://github.com/montjoie/lava-healthchecks.git
```

Comment:
Please move these to BayLibre github instead of your personal one.

Reply:
Done